### ISSUE TITLE
Speed up runtime

### ISSUE THREAD

**Issue by brianlangseth-NOAA** [brianlangseth-NOAA/Spatial-Workshop-SPASAM#37]
Adding this here more as a note than an issue. I noticed that running with selectivity estimated added a fair bit of time because these parameters are estimated in phase 4 (first two parameters) AND phase 5 (last two parameters). This, combined with F being estimated in phase 3 means that our F parameters get estimated 3 separate times.

We could speed up estimation by having the phase of all selectivity parameters be in a single phase.

**Comment by brianlangseth-NOAA** [comment]
Currently for me with first 105 years removed. 
Estimating Ndevs and F = 3.3 hrs. 
Estimating sel, rdevs, and F = 16 hrs
Estimating sel, rdevs, Ndev, and F = 20 hrs

**Comment by JDeroba** [comment]
@brianlangseth-NOAA I was able to speed up run time with a number of changes to data weighting. I made some other changes along the way, like changing phases, that did not really effect runtime, but I record them here so we have a proper record of changes. Using 7 fleets and nohess, these changes produced runtimes of ~1hour 40 minutes.
-selectivity parameters in phase 2; recruitment in phase 3; F in phase 4. All others in phase 1.
-OBS_yield_fleet_se_EM = 0.2 in all years with catch for all fleets
-OBS_catch_prop_N_EM (i.e., ESS of fishery catch) = 15 for all fleets in all years with catch, except fleet 5 (in a 7 fleet setup) which has ESS = 25. This difference in ESS for fleet 5 was necessary because selectivity was funky and comp fit poor. This slightly higher ESS solved those problems with little/no consequences elsewhere.
-Can turn estimation of init_abund back on and model works just fine now. (init_abund_switch change to 1; ph_init_abund change to 2).
-Change sel_beta3_start to 0.006 and sel_beta4_start to 0.1. This didn't really matter in the 7 fleet model except for a marginally better gradient, but it did in the 4 fleet attempts and these starting values just seem "better".
